readthedocs/payments/mixins.py

```diff
@@ -1,12 +1,12 @@
 """Payment view mixin classes."""
 
-from django.conf import settings
+from djstripe.models import APIKey
 
 
 class StripeMixin:
     """Adds Stripe publishable key to the context data."""
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        context["stripe_publishable"] = settings.STRIPE_PUBLISHABLE
+        context["stripe_publishable"] = APIKey.objects.filter(type="publishable").first().secret
```

Comment:
Type is defined as an enum (`djstripe.enums.APIKeyType`), we should use that instead of hardcoding the string.